tests/framework/extensions/clusters/dynamicSchema.go

```diff
@@ -0,0 +1,28 @@
+package clusters
```

Comment:
The comments on these structs are not very useful, and should either be removed or replaced with more informative comments.